web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -23,6 +23,12 @@ a.tabledrag-handle
 textarea
   max-width: 100%
 
+.section--sidebar-left #edit-join
```

Comment:
I removed this code, it is a leftover, because earlier there was a join collection button in the sidebar.